spring-cloud-dataflow-configuration-metadata/src/main/java/org/springframework/cloud/dataflow/configuration/metadata/ApplicationConfigurationMetadataResolverAutoConfiguration.java

```diff
@@ -29,6 +30,7 @@
  * @author Eric Bottard
  * @author Christian Tzolov
  */
+@AutoConfiguration
 @Configuration
```

Comment:
Should this configuration be removed?
`@AutoConfiguration`'s definition  contains `@Configuration(proxyBeanMethods = false)`.
Unless there is a reason to maintain it.   
I have seen this a couple times in the PR.

Reply:
Yep. Not sure why it was left in by OpenRewrite. I'll yank. Good catch.